src/DatatypeBindings.cpp

```diff
@@ -206,10 +234,72 @@ void DatatypeBindings::bind(pybind11::module& m){
+        .def_readwrite("autoFocusMode", &RawCameraControl::autoFocusMode)
+        .def_readwrite("lensPosition", &RawCameraControl::lensPosition)
         ;
 
+    // CameraControl enum bindings
+    py::enum_<RawCameraControl::AutoFocusMode>(rawCameraControl, "AutoFocusMode")
+        .value("OFF", RawCameraControl::AutoFocusMode::OFF)
+        .value("AUTO", RawCameraControl::AutoFocusMode::AUTO)
+        .value("MACRO", RawCameraControl::AutoFocusMode::MACRO)
+        .value("CONTINUOUS_VIDEO", RawCameraControl::AutoFocusMode::CONTINUOUS_VIDEO)
+        .value("CONTINUOUS_PICTURE", RawCameraControl::AutoFocusMode::CONTINUOUS_PICTURE)
+        .value("EDOF", RawCameraControl::AutoFocusMode::EDOF)
+    ;
+
+    py::enum_<RawCameraControl::AutoWhiteBalanceMode>(rawCameraControl, "AutoWhiteBalanceMode")
+        .value("OFF", RawCameraControl::AutoWhiteBalanceMode::OFF)
+        .value("AUTO", RawCameraControl::AutoWhiteBalanceMode::AUTO)
+        .value("INCANDESCENT", RawCameraControl::AutoWhiteBalanceMode::INCANDESCENT)
+        .value("FLUORESCENT", RawCameraControl::AutoWhiteBalanceMode::FLUORESCENT)
+        .value("WARM_FLUORESCENT", RawCameraControl::AutoWhiteBalanceMode::WARM_FLUORESCENT)
+        .value("DAYLIGHT", RawCameraControl::AutoWhiteBalanceMode::DAYLIGHT)
+        .value("CLOUDY_DAYLIGHT", RawCameraControl::AutoWhiteBalanceMode::CLOUDY_DAYLIGHT)
+        .value("TWILIGHT", RawCameraControl::AutoWhiteBalanceMode::TWILIGHT)
+        .value("SHADE", RawCameraControl::AutoWhiteBalanceMode::SHADE)
+    ;
+
+    py::enum_<RawCameraControl::SceneMode>(rawCameraControl, "SceneMode")
+        .value("UNSUPPORTED", RawCameraControl::SceneMode::UNSUPPORTED)
+        .value("FACE_PRIORITY", RawCameraControl::SceneMode::FACE_PRIORITY)
+        .value("ACTION", RawCameraControl::SceneMode::ACTION)
+        .value("PORTRAIT", RawCameraControl::SceneMode::PORTRAIT)
+        .value("LANDSCAPE", RawCameraControl::SceneMode::LANDSCAPE)
+        .value("NIGHT", RawCameraControl::SceneMode::NIGHT)
+        .value("NIGHT_PORTRAIT", RawCameraControl::SceneMode::NIGHT_PORTRAIT)
+        .value("THEATRE", RawCameraControl::SceneMode::THEATRE)
+        .value("BEACH", RawCameraControl::SceneMode::BEACH)
+        .value("SNOW", RawCameraControl::SceneMode::SNOW)
+        .value("SUNSET", RawCameraControl::SceneMode::SUNSET)
+        .value("STEADYPHOTO", RawCameraControl::SceneMode::STEADYPHOTO)
+        .value("FIREWORKS", RawCameraControl::SceneMode::FIREWORKS)
+        .value("SPORTS", RawCameraControl::SceneMode::SPORTS)
+        .value("PARTY", RawCameraControl::SceneMode::PARTY)
+        .value("CANDLELIGHT", RawCameraControl::SceneMode::CANDLELIGHT)
+        .value("BARCODE", RawCameraControl::SceneMode::BARCODE)
+    ;
+
+    py::enum_<RawCameraControl::AntiBandingMode>(rawCameraControl, "AntiBandingMode")
+        .value("OFF", RawCameraControl::AntiBandingMode::OFF)
+        .value("_50HZ", RawCameraControl::AntiBandingMode::_50HZ)
```

Comment:
THE_50_HZ rather? (As in camera resolution)

Reply:
Can also name it like `MAINS_50_HZ` to be more descriptive in this context.

Reply:
FWIW, I like that.

Reply:
Me too